### ISSUE TITLE
Bad performace

### ISSUE THREAD

**Issue by deeplearnerJHB** [qlan3/Explorer#3]
Hello, thanks for your released code. I run the code of averaged dqn and the performance is common. However, the performance of maxmin-dqn is really bad on MinAtar games. Whether can you reproduce the performance in the paper via this released code?

**Comment by qlan3** [comment]
Hi, could you please provide more info? Do you use exact code in Explorer? Do you do a grid search using the hyper-parameters provided in `configs/minatar.json` and pick the best hyper-parameter? How many runs do you have? Could you post your learning curves here?

**Comment by deeplearnerJHB** [comment]
Hi, I follow the configuration in configs/minatar.json except for your optimizer setting which is set as follow:

self.optimizer[0] = optim.RMSprop(self.Q_net[0].parameters(), lr=0.00025, alpha=0.95, centered=True, eps=0.01).

Because all my experiments including your dqn, ddqn and averaged-dqn can reach a good performance via this optimizer, I think it is also no problem for your maxmin-dqn.

Moreover, I set the number of Q-networks to 4 for training and haven't pay much attention to your grid search part which i think may cost a lot of time?

**Comment by qlan3** [comment]
I see. The setting of the optimizer and the number of Q-networks influence the performance greatly. It is better to do a hyper-parameter tuning before comparing different methods, otherwise it would be unfair. Grid search does take some time.  
If you tell me which MinAtar game your ran, I could tell you the best hyper-parameter **for the original code** I get to save you some time.

**Comment by JackZhangY** [comment]
Hi, thank you for your excellent work! I'm now trying to reproduce the results showed in Fig.4 of your paper. Could you offer the best hyperparameter combinations(mainly the number of Q-networks and learning rate) for each algorithm across different environments in README?  We can take it as a reference and save much time in hyperparameter tuning.

**Comment by deeplearnerJHB** [comment]
Hi, the MinAtar games I ran on are Breakout, Space Invaders, Asterix and Seaquest. Could you provide the best hyperparameter settings? Thank you very much.

**Comment by qlan3** [comment]
Hi @JackZhangY @deeplearnerJHB , here are the best hyper-parameters I get for Catcher, Lunarlander, and Pixelcopter.
However, somehow I cannot find logs for MinAtar, sorry about this! But I remember that for MinAtar games, the number of Q-networks is usually larger than 4. Again, sorry about this!
 
|  Catcher   | DQN  | DDQN  | Averaged DQN | Maxmin DQN |
|  ----  | ----  |  ----  | ----  |  ----  |
| lr  | 0.0001 | 0.0001 | 0.0001 | 0.0003 |
| target_networks_num  | / | / | 2 | 6 |


|  Lunarlander   | DQN  | DDQN  | Averaged DQN | Maxmin DQN |
|  ----  | ----  |  ----  | ----  |  ----  |
| lr  | 0.0003 | 0.0003 | 0.0003 | 0.001 |
| target_networks_num  | / | / | 4 | 3 |


|  Pixelcopter   | DQN  | DDQN  | Averaged DQN | Maxmin DQN |
|  ----  | ----  |  ----  | ----  |  ----  |
| lr  | 3e-05 | 3e-05 | 3e-05 | 0.0003 |
| target_networks_num  | / | / | 7 | 6 |
